docs: add product pricing for tappayments io module (#673)
taptap/tds-docs: .ci/hk/en/sdk/TapPayments/Global/features/product-pricing.mdx

---
title: Product Pricing
---

<head>
  <meta name="robots" content="noindex" />
</head>

> For global region pricing, we offer two pricing options for region pricing:

Reviewer: ```suggestion
We offer two options for setting up prices for different countries and regions:
```

> A. Use the system recommended prices

Reviewer: ```suggestion
A. Use the system-recommended prices
```

> B. Set your own price by country/region

Reviewer: ```suggestion
B. Set your own prices by country/region
```

> * Please note that different currencies have different decimal precision rules, please complete this table.

Reviewer: ```suggestion
* Please note that different currencies have different decimal precision rules. See the table below for more information.
```
